cdc/owner.go

```diff
@@ -151,6 +154,7 @@ func NewOwner(
 		cfRWriter:               cli,
 		etcdClient:              cli,
 		gcTTL:                   gcTTL,
+		gcBLT:                   gcBLT,
```

Comment:
may be `maxGCSafeTimeDelay` is a better name?

Reply:
or `maxGCBlockTime`

Reply:
We do not promote the use of abbreviations that are not conventional

Reply:
Thanks, I will change it name to `maxGCBlockTime`. It is a better name.